app/components/vote-overview.hbs

```diff
@@ -1,6 +1,12 @@
-<C3Chart
-  @data={{this.data}}
-  @color={{this.color}}
-  @pie={{this.pie}}
-  @legend={{this.legend}}
-/>
+<div class="c-chart">
```

Comment:
Looks good 👍 The only thing missing seems to be the hover effect that the previous chart did. I don't really think it's needed, but Jodi might need to decide if it's acceptable like this?